src/com/thizthizzydizzy/treefeller/compat/MMOCoreCompat.java

```diff
@@ -236,6 +247,152 @@ public String writeToConfig(HashMap<String, Double> value){
+                List l = (List)o;
+                HashMap<String, Integer> professions = new HashMap<>();
+                for(Object lbj : l){
+                    if(lbj instanceof Map){
+                        Map m = (Map)lbj;
+                        for(Object obj : m.keySet()){
+                            String profession = null;
+                            if(obj instanceof String){
+                                profession = (String)obj;
+                            }
+                            if(profession==null)continue;
+                            Integer lvlreq = Option.loadInt(m.get(obj));
+                            if(lvlreq==null)continue;
+                            if(professions.containsKey(profession)){
+                                professions.put(profession, professions.get(profession)+lvlreq);
+                            }else{
+                                professions.put(profession, lvlreq);
+                            }
+                        }
+                    }
+                }
+                return professions;
+            }
+            return null;
+        }
+        @Override
+        public ItemBuilder getConfigurationDisplayItem(HashMap<String, Integer> value){
+            return new ItemBuilder(Material.GOLDEN_PICKAXE);
+        }
+        @Override
+        public void openGlobalModifyMenu(MenuGlobalConfiguration parent){
+            parent.open(new MenuModifyStringIntegerMap(parent, parent.plugin, parent.player, name, 0, Integer.MAX_VALUE, false, false, globalValue, ( value) -> {
+                globalValue = value;
+            }));
+        }
+        @Override
+        public void openToolModifyMenu(MenuToolConfiguration parent, Tool tool){
+            parent.open(new MenuModifyStringIntegerMap(parent, parent.plugin, parent.player, name, 0, Integer.MAX_VALUE, true, false, toolValues.get(tool), (value) -> {
+                if(value==null)toolValues.remove(tool);
+                else toolValues.put(tool, value);
+            }));
+        }
+        @Override
+        public void openTreeModifyMenu(MenuTreeConfiguration parent, Tree tree){
+            parent.open(new MenuModifyStringIntegerMap(parent, parent.plugin, parent.player, name, 0, Integer.MAX_VALUE, true, false, treeValues.get(tree), (value) -> {
+                if(value==null)treeValues.remove(tree);
+                else treeValues.put(tree, value);
+            }));
+        }
+        @Override
+        public String writeToConfig(HashMap<String, Integer> value){
+            if(value==null)return "";
+            String s = "{";
+            String str = "";
+            for(String st : value.keySet()){
+                str+=", "+st+": "+value.get(st);
+            }
+            if(!str.isEmpty())s+=str.substring(2);
+            return s+"}";
+        }
+        @Override
+        protected DebugResult doCheck( TreeFeller plugin, Tool tool, Tree tree, Block block, Player player, ItemStack axe ){

```

Comment:
Since options aren't run through the compatibility system, this will still run when the plugin isn't installed